examples/core-triangle/App.zig

```diff
@@ -24,64 +25,95 @@ pub fn init(
     core.on_tick = app_mod.id.tick;
     core.on_exit = app_mod.id.deinit;
 
-    // Create our shader module
-    const shader_module = core.device.createShaderModuleWGSL("shader.wgsl", @embedFile("shader.wgsl"));
-    defer shader_module.release();
-
-    // Blend state describes how rendered colors get blended
-    const blend = gpu.BlendState{};
-
-    // Color target describes e.g. the pixel format of the window we are rendering to.
-    const color_target = gpu.ColorTargetState{
-        .format = core.windows.get(core.main_window, .framebuffer_format).?,
-        .blend = &blend,
-    };
-
-    // Fragment state describes which shader and entrypoint to use for rendering fragments.
-    const fragment = gpu.FragmentState.init(.{
-        .module = shader_module,
-        .entry_point = "frag_main",
-        .targets = &.{color_target},
-    });
-
-    // Create our render pipeline that will ultimately get pixels onto the screen.
-    const label = @tagName(mach_module) ++ ".init";
-    const pipeline_descriptor = gpu.RenderPipeline.Descriptor{
-        .label = label,
-        .fragment = &fragment,
-        .vertex = gpu.VertexState{
+    const main_window = core.windows.getValue(core.main_window);
+    if (main_window.native != null) {
```

Comment:
what is this trying to guard against?

Reply:
will remove, no need for it, should have been removed earlier